app/views/games/show.html.haml

```diff
@@ -109,8 +108,8 @@
       - prevscore = -1
       - currentnum = 1
       - countnum = 1
-      -all_players = @game.registrations.group_by(&:display_faction_id).sort_by{|x| x.display_score}.reverse_each do |p|
-    %div.humans.clear{:style=>"float:left;width:40%"}
+      -all_players = @game.registrations.group_by(&:display_faction_id).sort_by{|x| x.display_score}
```

Comment:
This won't work, let's trace through what it's going to do:
`@game.registrations` returns an array of a bunch of players.
`.group_by(&:display_faction_id)` transforms this array into a hashmap, of faction_id => [list of players]
Now we're calling `.sort_by` on this hashmap, but we really want to sort the lists of players within the hashmap, which are just the values.
When we do a group_by, the order is retained, so we can simplify this by just doing the sort before we do the grouping, so:
`@game.registrations.sort_by(&:display_score).group_by(&:display_faction_id)`